lib/src/source/hosted.dart

```diff
@@ -62,10 +62,10 @@ class HostedSource extends Source {
     var url = io.Platform.environment['PUB_HOSTED_URL'];
     if (url == null) return null;
     var uri = Uri.parse(url);
-    if (uri.scheme?.isEmpty ?? true) {
+    if (!['http', 'https'].contains(uri.scheme.toLowerCase())) {
```

Comment:
Why not `uri.hasScheme('http') || uri.hasScheme('https')`

Reply:
I mean: `uri.isScheme('http') || uri.isScheme('https')`

Reply:
https://api.dart.dev/stable/2.10.3/dart-core/Uri/isScheme.html

Reply:
Ah - that works!

Reply:
Done